lib/parallel_cucumber/hooks.rb

```diff
@@ -0,0 +1,20 @@
+module ParallelCucumber
+  class Hooks
+
+    @after_batch_hooks ||= []
+
+    class << self
+
+      def after_batch(&proc)
+        raise 'Please provide a valid callback' unless proc.respond_to?(:call)
+        @after_batch_hooks << proc
+      end
+
+      def fire_after_batch_hooks(*args)
```

Comment:
Args are not variable. Please specify which args will be passed (batch_results, batch_id, env), preferable using named arguments.